bin/webindex

```diff
@@ -19,6 +19,9 @@ export WI_HOME=$( cd "$( dirname "$BIN_DIR" )" && pwd )
 
 if [ -f $WI_HOME/conf/webindex-env.sh ]; then
   . $WI_HOME/conf/webindex-env.sh
+else
+  test -z "$WI_EXECUTOR_INSTANCES" && export WI_EXECUTOR_INSTANCES=1
```

Comment:
Couldn't these two lines could always be done?  Could be a case where webindex-env.sh exists but does not set those.